build

```diff
@@ -9,8 +9,8 @@ then
 fi
 
 [ -z "$JAVA_PATH" ] && JAVA_PATH=java
-[ -z "$APKTOOL_PATH" ] && APKTOOL_PATH=/opt/apktool/apktool.jar
-[ -z "$UBER_APK_SIGNER_PATH" ] && UBER_APK_SIGNER_PATH=/opt/uber-apk-signer/uber-apk-signer-1.2.1.jar
+[ -z "$APKTOOL_PATH" ] && APKTOOL_PATH=/opt/bttv/apktool_2.9.3.jar
```

Comment:
revert this

Reply:
How can I update the APKTool version inside the container image ? It needs to be 2.9.3 now.